packages/components/src/templates/next/layouts/Collection/CollectionClient.tsx

```diff
@@ -1,402 +1,153 @@
-                      filteredItems.length === 1 ? "result" : "results"
-                    }`
-                  : `${items.length} ${
-                      items.length === 1 ? "article" : "articles"
-                    }`}
-                {searchValue !== "" && (
-                  <>
-                    {" "}
-                    for "<b>{searchValue}</b>"
-                  </>
-                )}
-              </p>
-              {/* Commenting out applied filter display temporarily
-              {appliedFilters.length > 0 && (
-                <div className="flex flex-row flex-wrap gap-3">
-                  {getAppliedFiltersWithLabels(filters, appliedFilters).map(
-                    (appliedFilter) => (
-                      <Pill
-                        key={`${appliedFilter.appliedFilterTypeId}-${appliedFilter.appliedFilterId}`}
-                        content={appliedFilter.label}
-                        onClose={() =>
-                          updateAppliedFilters(
-                            appliedFilters,
-                            setAppliedFilters,
-                            appliedFilter.appliedFilterTypeId,
-                            appliedFilter.appliedFilterId,
-                          )
-                        }
-                      />
-                    ),
+        <div className={compoundStyles.content()}>
+          <div className="flex w-full flex-col gap-3">
+            <div className="flex w-full flex-col justify-between gap-x-6 gap-y-2 md:flex-row">
+              <div className="flex h-full w-full items-center gap-3">
+                <p className="prose-headline-lg-regular text-base-content-medium">
+                  {appliedFilters.length > 0 || searchValue !== ""
+                    ? `${filteredCount} search result${
+                        filteredCount === 1 ? "" : "s"
+                      }`
+                    : `${items.length} article${items.length === 1 ? "" : "s"}`}
+                  {searchValue !== "" && (
+                    <>
+                      {" "}
+                      for "<b>{searchValue}</b>"
+                    </>
                   )}
-                </div>
-              )}*/}
+                </p>
+              </div>
             </div>
-            <div className="w-full shrink-0 sm:w-[260px]">
-              <CollectionSort
-                sortBy={sortBy}
-                setSortBy={setSortBy}
-                sortDirection={sortDirection}
-                setSortDirection={setSortDirection}
-              />
+            <div className="flex h-full w-full flex-col gap-0">
+              {paginatedItems.length > 0 &&
+                paginatedItems.map((item) => (
+                  <CollectionCard
+                    key={Math.random()}
+                    {...item}
+                    LinkComponent={LinkComponent}
+                  />
+                ))}
+              {paginatedItems.length === 0 && searchValue !== "" && (
```

Comment:
issue: I think this condition is too specific. If the user solely filtered using the filters on the left side and `paginatedItems.length` is 0, then the message does not appear.

<img width="1199" alt="Screenshot 2024-08-16 at 10 43 19" src="https://github.com/user-attachments/assets/b2b3b2d3-6073-4ebd-a22f-f0f76c877bce">

Reply:
never touch filter behavior here. This PR solely for layout changes hehe.

I think @sehyunidaaa is working on empty screen styling, will fix alongside that PR.